data_query.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX animal_idx ON visits (animal_id);
+CREATE INDEX vet_idx ON visits (vet_id);
+CREATE INDEX owners_idx ON owners (email);
```

Comment:
[OPTIONAL]

This is a nice-to-have a suggestion as it won't improve the query that was reported as problematic in the requirements for this project.

As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:

```
 CREATE INDEX ON owners ((lower(email)));
```